install.sh

```diff
@@ -109,7 +112,8 @@ main() {
     VERSION \
     OPTIONAL \
     ALL_FLAGS \
-    SSH_ARGS
+    RSH_ARGS \
+    RSH
```

Comment:
Looks like we need to set the default for `RSH` to `ssh`.

Reply:
Embarrassing. Fixed!